packages/panzer/adapters-stk/test/ip_coordinates/CMakeLists.txt

```diff
@@ -9,10 +9,15 @@ INCLUDE_DIRECTORIES(${${PACKAGE_NAME}_SOURCE_DIR}/test/bcstrategy)
 SET(UNIT_TEST_DRIVER
   ${PANZER_UNIT_TEST_MAIN})
 
+IF(PANZER_HAVE_EPETRA)
+  SET(SOURCES epetra_ip_coordinates.cpp)
+ELSE(PANZER_HAVE_EPETRA)
+  SET(SOURCES tpetra_ip_coordinates.cpp)
+ENDIF(PANZER_HAVE_EPETRA)
```

Comment:
I think the tpetra test should always be enabled (not disabled if epetra is enabled).

Reply:
Ok changed, now Tpetra test will always be enabled, only epetra will be optional